src/components/Context/ContextProvider.js

```diff
@@ -85,11 +85,18 @@ const fetchBifiPrice = async ({ setBifiPrice, setMarketCap }) => {
 const fetchEarnings = async ({ setDailyEarnings, setTotalEarnings }) => {
   let earnings = await getEarnings() || { daily: 0, total: 0 };
   if(!earnings.total){ earnings.total = 0; }
+  if (typeof earnings.total === "string" || earnings.total instanceof String) {

```

Comment:
fix for API returning bignumber string.
not needed with this update https://github.com/beefyfinance/beefy-api/pull/73/files